src/lasdi/latent_dynamics/edmd.py

```diff
@@ -0,0 +1,172 @@
+import numpy as np
+import torch
+from scipy.integrate import odeint
+from . import LatentDynamics
+from ..inputs import InputParser
+from ..fd import FDdict
+from scipy.linalg import logm
+import importlib 
+
+def get_function_from_string(func_str):
+    # Split the string into module and function
+    module_name, func_name = func_str.rsplit('.', 1)
+    module = importlib.import_module(module_name)
+    return getattr(module, func_name)
+
+class edmd(LatentDynamics):
+    fd_type = ''
+    fd = None
+    fd_oper = None
+
+    def __init__(self, dim, high_order_terms, rand_functions, nt, config):
```

Comment:
Based on the usage shown below, `high_order_terms` and `rand_functions` can be parsed from `config` and need not be passed as input arguments. Can we parse them from `config` and reduce the number of input arguments?